Formula/lima.rb

```diff
@@ -0,0 +1,29 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima.git",
```

Comment:
Can we use a tarball instead of a Git clone? `brew` can parse the version from the tarball URL and pass this to `go build`.

Reply:
The current Makefile depends on `git describe` to detect the version number.

https://github.com/AkihiroSuda/lima/blob/c82737b29cf54225c3ec8ec7073609d42924017c/Makefile#L10

Reply:
Yes, that's why I suggested

> `brew` can parse the version from the tarball URL and pass this to `go build`.

Reply:
Having to do a Git clone just to be able to do `git describe` is really excessive and unnecessary.

Reply:
Passing a version string like `make VERSION=1.2.3` seems already work